desktop-app/app/components/Header/index.js

```diff
@@ -12,12 +12,17 @@ import NavigationControlsContainer from '../../containers/NavigationControlsCont
 import BookmarksBar from '../../containers/BookmarksBarContainer';
 import AppNotification from '../AppNotification/AppNotification';
 import Logo from '../icons/Logo';
+import ZenButton from '../ZenButton';
 
-const Header = () => {
+const Header = props => {
   const classes = useStyles();
 
   return (
-    <div className={classes.container}>
+    <div
+      className={`${classes.container} ${
```

Comment:
@sidthesloth92 Can you please make use of the `classnames` module here? 
That will make it more readable.

Reply:
Sure will do. Wasn't aware of the module.. :D

Reply:
Done.